### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Jeph53707**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,34 @@
-import logo from './logo.svg';
+
 import './App.css';
+import Form from "./components/Form.js";
+
+
+
```
Comment (allan-viveiros) [inline src/App.js]: I think is a good practice to remove the commented code before submit the project

File: src/components/Form.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        // console.log ("tt")
+        console.log (e.target.value)
+        setSelectedValue (e.target.value)
+
+
+    }
+
+    const [art, setArt] = useState ([]);
+
+    // const [artOrientation, setArtOrientation] = useState (null)
```
Comment (seldaerkilic) [inline src/components/Form.js]: if this is not in use, you may clear it before merge

**Review by seldaerkilic — COMMENTED** [review]
Well done! Congrats

File: src/components/Form.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+         
+         
+        },
+      }).then((response) => {
+        console.log(response.data.artObjects);
+        setArt (response.data.artObjects)
+       
+      })},[selectedValue]);
+
+
```
Comment (allan-viveiros) [inline src/components/Form.js:67]: I think is a good practice to use the HTMLfor on labels to make it easy for screen readers

Comment (allan-viveiros) [reply src/components/Form.js:67]: And have and id on the input/select t o connect with the label

**Pull request merged**